Xamarin.Forms.Core/OnAppTheme.cs

```diff
@@ -2,29 +2,14 @@
 
 namespace Xamarin.Forms
 {
-	public class OnAppTheme<T> : BindingBase
+	class OnAppTheme<T> : BindingBase
```

Comment:
I don't think we want to expose this as-is, as it has little value unless used as a Binding. Keeping this would be confusing, and would led to customers no getting the intended behavior.